### PR TITLE
Allow to filter by multiple publishers

### PR DESCRIPTION
**Pull request by hugovk**
Resolves https://github.com/City-of-Helsinki/linkedevents/issues/204

Allow to filter by multiple comma-separated `publisher` parameters, in the same way as `data_source`.

Note: untested, but CI passes.

### CODE REVIEW

**Comment by juyrjola** [comment]
Looks good, thanks!

**Comment by hugovk** [comment]
Great! Do you know when this will likely be deployed?

eg. https://api.hel.fi/linkedevents/v1/event/?publisher=ahjo:46104,ahjo:46111

File: events/api.py [hunk 1]
```diff
@@ -1360,8 +1360,10 @@ def _filter_event_queryset(queryset, params, srs=None):
         cond = 'end_time - start_time >= %s :: interval'
         queryset = queryset.extra(where=[cond], params=[str(dur)])
 
+    # Filter by publisher, multiple sources separated by comma
     val = params.get('publisher', None)
     if val:
+        val = val.split(',')
         queryset = queryset.filter(publisher__id=val)
```
Comment (akx) [inline events/api.py]: Shouldn't this be `publisher_id__in` now?

**Comment by juyrjola** [comment]
There was a small bug in the patch, but I fixed that now and deployed it to production.

File: events/api.py (hunk 1 above)
Comment (juyrjola) [inline events/api.py]: Yup, it should.

Comment (akx) [inline events/api.py]:
```python
>>> from events.models import Event
>>> qs = Event.objects.filter(publisher__id=['ee', 'qq', 'yy'])
>>> print(qs.query)
SELECT LOTS_OF_COLUMNS FROM "events_event" WHERE "events_event"."publisher_id" = ['ee', 'qq', 'yy'] ORDER BY "events_event"."tree_id" ASC, "events_event"."lft" ASC
>>> qs = Event.objects.filter(publisher_id__in=['ee', 'qq', 'yy'])
>>> print(qs.query)
SELECT LOTS_OF_COLUMNS FROM "events_event" WHERE "events_event"."publisher_id" IN (ee, qq, yy) ORDER BY "events_event"."tree_id" ASC, "events_event"."lft" ASC
```

**Comment by hugovk** [comment]
Thanks all for the quick response!

**Pull request merged**